### DISCUSSION TITLE
Sidebar with logo on small landscape screens is very cramped

### DISCUSSION THREAD

**Discussion by dgw** [pradyunsg/furo#683] [Potential Issue]
I realize the severity of this "problem" will vary depending on the exact project, logo design, image dimensions, specific mobile device, etc., but for a project I'm on that's planning to switch to Furo for our next major release later this year:

![Screenshot_20230716-144931173](https://github.com/pradyunsg/furo/assets/164140/637067dd-b3ce-4c06-a9a3-c691e29535c4)

That tiny scrollable area is hardly enough to properly navigate the documentation, I'm sure you'll agree. I just noticed it today reviewing a documentation PR on my phone (Asus ROG Phone 3).

I'm sure we can come up with custom CSS workarounds. We can hide the logo for viewports with less than _n_ vertical space, for example. But I'm curious if this is something you might have a better solution for if it's baked into the theme. I'm sure we could locally override which region of the sidebar is scrollable, too, but that seems like getting into something that should be considered at the theme level.

This is the first time I've noticed something in Furo that I thought could be handled better, and it's rather an edge case. (Who actually reads from a phone in landscape, right??) Overall we are extremely pleased with the theme, and very excited to switch from Alabaster!

----

_Note: We aren't using the very latest Furo package, but I did make sure to check the demo site at pradyunsg.me/furo and it *seems* that the sidebar scroll behavior/layout hasn't changed. Furo's docs just have a much shorter masthead in the sidebar, with no logo. 😁_

**Comment by pradyunsg** [comment]
What's the screen resolution?

**Reply by dgw** [reply]
2340 x 1080
